### DISCUSSION TITLE
Block Filter seems to have stopped working

### DISCUSSION THREAD

**Discussion by lucasmariani** [ProxymanApp/Proxyman#2147] [General]
I use Proxyman with Atlantis and for the past couple of releases I have been unable to get the block list feature working. I have several rules in the block list, they are all set to enable but hide. However, I can still see all of the requests.

Also, and unrelated, I have the app pinned to my favorites, and switching between the app listed under Pinned and the app under my device under Atlantis causes the list of requests to disappear, even though they are the same app. I can only see the requests under Atlantis, but when clicking on the app in the Pinned list, all requests are removed.

**Comment by NghiaTranUIT** [comment]
Block List doesn't work with Atlantis. 

Atlantis directly captures your traffic on your iOS App, and sent to Proxyman app to preview.

If you'd like to use the Block List, you should use the normal HTTP Proxy. You can find the Guide at Certificate Menu -> Install for iOS -> Physical Device or Simulator 👍 

> Also, and unrelated, I have the app pinned to my favorites, and switching between the app listed under Pinned and the app under my device under Atlantis causes the list of requests to disappear, even though they are the same app.

From our design, the app in Pin is a physical app on your iOS Simulator (local on your Mac), and the app on Atlantis is different. So, Atlantis traffic doesn't appear on the Pin app.

It's a bug, it should be the same if they have the same bundle ID
